packages/math-input/src/components/keypad-legacy/two-page-keypad.tsx

```diff
@@ -14,8 +14,7 @@ import {
     innerBorderWidthPx,
     offBlack16,
 } from "../common-style";
-import Tabbar from "../tabbar/tabbar";
-import {TabbarItemType} from "../tabbar/types";
+import Tabbar, {TabbarItemType} from "../tabbar";
```

Comment:
TypeScript is annoying ambiguous on whether you need to import using `import type` vs just `import` for types... except when it matters. For the time being, I think it's safer to use `import type` whenever we're importing an actual type. 

```suggestion
import Tabbar, {type TabbarItemType} from "../tabbar";
```

It might even make sense to move the import of the `TabbarItemType` down to an `import type` section below line 21.